domain/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Why do we need this? Prefer to remove.

Reply:
I had kept this because mockito doesnt allow mocking final classes unless you do this and all kotlin classes are final by default. But since I dont need to use mockito anymore I removed it